jax/_src/nn/functions.py

```diff
@@ -203,6 +204,7 @@ def sigmoid(x: ArrayLike) -> Array:
     :func:`log_sigmoid`
 
   """
+  numpy_util.check_arraylike("sigmoid", x, emit_warning=True)
```

Comment:
I think removing `emit_warning=True` is safe here, becuase `lax.logistic` requires array-like input.